x-pack/libbeat/licenser/elastic_fetcher_integration_test.go

```diff
@@ -27,8 +27,8 @@ func getTestClient() *elasticsearch.Client {
 	client, err := elasticsearch.NewClient(elasticsearch.ClientSettings{
 		URL:              host,
 		Index:            outil.MakeSelector(),
-		Username:         cli.GetEnvOr("ES_USER", ""),
-		Password:         cli.GetEnvOr("ES_PASS", ""),
+		Username:         "myelastic", // NOTE: I will refactor this in a followup PR
+		Password:         "changeme",
```

Comment:
@tsg, @andrewkroh I will do a followup on this, I didn't want to waste time on docker-compose environment variable and slow the merge. Since we need to add integration test for this new scenario we can fix it at the same time.